setup.py

```diff
@@ -41,8 +41,8 @@ def is_pure(self):
 ray_deps = ["ray>=1.0.0,<1.2.0", "pyarrow==1.0"]
 remote_deps = ["rpyc==4.1.5", "cloudpickle==1.4.1", "boto3==1.4.8"]
 spreadsheet_deps = ["modin-spreadsheet>=0.1.0"]
-
-all_deps = dask_deps + ray_deps + remote_deps + spreadsheet_deps
+sql_deps = ["dfsql==0.2.2"]
```

Comment:
How quickly will this evolve? Is pinning to 0.2.2 a good idea or should we have a `>=` so we don't need to bump every release?

Reply:
I think you are right, should be a good idea to use `>=`